### DISCUSSION TITLE
开发者问卷：访客首页，库存显示具体数量？还是保持当前状态？

### DISCUSSION THREAD

**Discussion by Baiyuetribe** [Baiyuetribe/kamiFaka#21] [开发者计划--开发日志]
![image](https://user-images.githubusercontent.com/43716063/101135755-341fa880-3647-11eb-8918-bbc059bc055f.png)

**Comment by yangyzp** [comment]
个人觉得显示具体数量好点。（当然，如果后台能切换显示模式最好，满足各类人需求）
